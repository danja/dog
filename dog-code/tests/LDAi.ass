TEST
LDAi 66
DUMP
