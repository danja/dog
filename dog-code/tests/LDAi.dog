F9 TEST
10 LDAi 66
66
F8 DUMP
