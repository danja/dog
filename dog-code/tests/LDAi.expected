{
"pc": 1,
"status": 0,
"accA": 102,
"accB": -1,
"xReg": -1,
"pcStackP": -1,
"xStackP": -1
}
